README.md

```diff
@@ -13,13 +13,20 @@
   * [Paciello Group](https://developer.paciellogroup.com/resources/)
   * [Ebay accessibility patterns](https://ebay.gitbooks.io/mindpatterns/content/)
   * [BBC accessibility guidelines](http://www.bbc.co.uk/guidelines/futuremedia/accessibility/)
+  * [Vox Media Accessibility Checklist](http://accessibility.voxmedia.com/)
 
-## Blogs, etc.
+## Other Resources (Blogs, Individuals, Videos, etc)
```

Comment:
Good change! I think we can actually get rid of the parentheses and what's within them, to make it more concise.